bin/bump-version

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Automatically updates the tap for a newly-released version of materialized.
+
+set -euo pipefail
+
+if [[ $# -ne 1 ]]; then
+  echo "usage: $0 VERSION" >&2
+  exit 1
+fi
+
+cd "$(dirname "$0")/.."
+
```

Comment:
could we check that `~/.netrc` exists and is configured at least vaguely correctly (`grep bintray $HOME/.netrc"`) here?

Reply:
(obviously only if we're not in CI)

Reply:
👍 the beauty of the bump-version script is that it never runs in CI

Reply:
(going to address in a follow-up)